src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return 80;
+  } else if (days >= 3 && days < 6) {
+    return 100;
+  } else if (days === 6) {
+    return 220;
```

Comment:
This condition is unnecessary because the same discount applies for 3 or more days, including the 6th day. You can combine this logic with the previous `else if` condition.